.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.0
+
+jobs:
+  "Test":
+    docker: # Replace this with a fastlane owned docker image at some point
+      - image: joshdholtz/fastlane-docs:0.0.1
+    environment:
+      LC_ALL: C.UTF-8
+      LANG: C.UTF-8
+    working_directory: ~/code
+    steps:
+      - checkout
+      - run: pip3 install pipenv
+      - run: pipenv install
+      - run: bundle update
+      - run: pipenv run bundle exec fastlane test
```

Comment:
In the old one this command was not run via `pipenv run` - what's the difference?

Reply:
We need access to `mkdocs` and `mkdocs` is only available inside of a `pipenv shell`. Since are we in a CircleCI shell already we can't enter a `pipenv shell` so we have to do it on a command by command basis. Without `pipenv run` it errors with `mkdocs not found`.

Reply:
Ah, I remember those from when I tried to migrate. Pain in the ass.